conf/st2rc.sample.ini

```diff
@@ -22,6 +22,15 @@ password = testpassword
 
 [api]
 url = http://127.0.0.1:9101/v1
+# or from remote
+# url = http://127.0.0.1/api/v1
```

Comment:
I assume here it should be some other external domain / IP and not, 127.0.0.1, right?

Perhaps we should use example.com?